### PR TITLE
[Netmanager] LOGGING USERS WHO PERFORM DATA EXPORTS

### PR DESCRIPTION
**Pull request by Codebmk**
#### Summary of Changes (What does this PR do?)

- Added service to data export api headers to enable logging of user activity

#### Status of maturity (all need to be checked before merging):

- [ ] I've tested this locally
- [ ] I consider this code done
- [ ] This change ready to hit production in its current state

#### How should this be manually tested?

- Please include the steps to be done inorder to setup and test this PR.

#### What are the relevant tickets?
- [AN-376](https://airqoteam.atlassian.net/browse/AN-376) 

#### Screenshots (optional)

[AN-376]: https://airqoteam.atlassian.net/browse/AN-376?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: netmanager/src/views/apis/analytics.js [hunk 1]
```diff
@@ -18,7 +18,13 @@ export const getSitesApi = async () => {
 };
 
 export const downloadDataApi = async (data) => {
-  return axios.post(DOWNLOAD_CUSTOMISED_DATA_URI, data).then((response) => response.data);
+  const headers = {
+    service: 'data-export'
+  };
+
+  return axios
```
[the remark this answers is not in the document]
Comment (Baalmart) [reply netmanager/src/views/apis/analytics.js:25]: Also, please note that this requires some tests to confirm that the logs are being captured correctly. You can use postman for this and then later perform some frontend tests.

From what I see here, we are not even using the `Authorization` header so I am not sure if things will work. We need to ensure that data export is happening on a `protected` endpoint

[the remark this answers is not in the document]
Comment (Baalmart) [reply netmanager/src/views/apis/analytics.js:25]: @Codebmk , please coordinate with the author/maintainer of the analytics microservice for more assistance on their V2 migration plan. Thanks!

[the remark this answers is not in the document]
Comment (Codebmk) [reply netmanager/src/views/apis/analytics.js:25]: Yes @Baalmart I have added a new ENV called `REACT_APP_DOWNLOAD_URL`

[the remark this answers is not in the document]
Comment (Codebmk) [reply netmanager/src/views/apis/analytics.js:24]: This was tested out yesterday and was working fine. However, the deploy preview routes seem to be down. Can't login.
image

**Review by Baalmart — APPROVED** [review]
Thanks @Codebmk , this LGTM.

**Pull request merged**